### DISCUSSION TITLE
Directory Server update

### DISCUSSION THREAD

**Discussion by softins** [jamulussoftware/jamulus#2169] [Announcements]
Just a heads-up: sometime on Sunday evening (12 Dec) European time, the three directory servers for Any Genres (1, 2 and 3) will be moving to a new server host, with a different IP address.

The DNS entries for them will be updated at that time, but due to DNS caching and propagation, it may take up to an hour or more for clients and servers to obtain the new IP address, and during that time clients may not see all of the servers normally registered in each directory.

We will announce when the change has been done. Any server operators might want to check that their server continues to appear in the list at [Jamulus Exporer](https://explorer.jamulus.io), and restart their server if not.

> **UPDATE: This change was brought forward to Sunday afternoon, and has been completed without issue. The new directory servers now contain the expected lists of servers and the old ones are empty, indicating that all servers have picked up the new IP address correctly and re-registered.**

The affected directory server names are:

* Current: **anygenre1.jamulus.io**, **anygenre2.jamulus.io**, **anygenre3.jamulus.io**

* Deprecated: **jamulusanygenre1.fischvolk.de**, **jamulusanygenre2.fischvolk.de**, **jamulusanygenre3.fischvolk.de**, **jamulus.fischvolk.de**, **jamulusallgenres.fischvolk.de**

The deprecated names will be updated and maintained for backward compatibility with old versions of Jamulus, but server operators are encouraged to update their configuration to use the current names, as the old names might be removed in the future.

**Comment by pljones** [comment]
Also note that anyone who was using **jamulusanygenre2.fischvolk.de** or **jamulusallgenres.fischvolk.de** without specifying a port number will find their server now is listed on **anygenre1.jamulus.io**.  (The same applies to anyone using the "private" directory at fischvolk.de.)

To list under Any Genre 2, use **anygenre2.jamulus.io:22224** (i.e. the port number _must_ be specified).

(A "private" directory is also available at jamulus.io on the default port.)

**Comment by softins** [comment]
This change has now been completed - see the update above in the original announcement.

**Comment by ann0see** [comment]
Since this has been completed, I'll unpin this announcement. If anything went wrong, feel free to open a new discussion thread.
